

This a model from the article:  
** Transient heterogeneity in extracellular protease production by Bacillus subtilis. **   
Veening JW, Igoshin OA, Eijlander RT, Nijland R, Hamoen LW, Kuipers OP _Mol.
Syst. Biol._ 2008 ; Volume: 4 : 184
[18414485](http://www.ncbi.nlm.nih.gov/pubmed/18414485),  
**Abstract:**   
The most sophisticated survival strategy Bacillus subtilis employs is the
differentiation of a subpopulation of cells into highly resistant endospores.
To examine the expression patterns of non-sporulating cells within
heterogeneous populations, we used buoyant density centrifugation to separate
vegetative cells from endospore-containing cells and compared the
transcriptome profiles of both subpopulations. This demonstrated the
differential expression of various regulons. Subsequent single-cell analyses
using promoter-gfp fusions confirmed our microarray results. Surprisingly,
only part of the vegetative subpopulation highly and transiently expresses
genes encoding the extracellular proteases Bpr (bacillopeptidase) and AprE
(subtilisin), both of which are under the control of the DegU transcriptional
regulator. As these proteases and their degradation products freely diffuse
within the liquid growth medium, all cells within the clonal population are
expected to benefit from their activities, suggesting that B. subtilis employs
cooperative or even altruistic behavior. To unravel the mechanisms by which
protease production heterogeneity within the non-sporulating subpopulation is
established, we performed a series of genetic experiments combined with
mathematical modeling. Simulations with our model yield valuable insights into
how population heterogeneity may arise by the relatively long and variable
response times within the DegU autoactivating pathway.

  

This model originates from BioModels Database: A Database of Annotated
Published Models. It is copyright (c) 2005-2009 The BioModels Team.  
For more information see the [terms of
use](http://www.ebi.ac.uk/biomodels/legal.html).  
To cite BioModels Database, please use [Le Nov√®re N., Bornstein B., Broicher
A., Courtot M., Donizelli M., Dharuri H., Li L., Sauro H., Schilstra M.,
Shapiro B., Snoep J.L., Hucka M. (2006) BioModels Database: A Free,
Centralized Database of Curated, Published, Quantitative Kinetic Models of
Biochemical and Cellular Systems Nucleic Acids Res., 34: D689-D691.](http://ww
w.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&pubmedid=16381960)

